kotori/daq/storage/influx.py

```
@@ -71,6 +71,15 @@ def query(self, expression: str, tdata: Munch = None):
         return self.influx_client.query(expression)
 
     def write(self, meta, data):
+        if isinstance(data, dict):
+            self.write_single(meta, data)
+        elif isinstance(data, list):
+            for item in data:
+                self.write_single(meta, item)
+        else:
+            raise ValueError(f"Type of data {type(data)} not accepted")
```

Ruff (0.15.0)

[warning] Prefer `TypeError` exception for invalid type

(TRY004)